### PR TITLE
feat(cost-query-set): create managed cost-query-set & edit cost-query…

### PR DESCRIPTION
**Pull request by piggggggggy**
…-store name

### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [x] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

- Create managed cost-query-sets
- Add cost-query menuSet in CostExplorer LNB
- edit cost-query-store naming
image

### CODE REVIEW

File: apps/web/src/services/cost-explorer/cost-analysis/config.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+export const managedCostQuertSetIdList: string[] = [
+    MANAGED_COST_QUERY_SET_IDS.MONTHLY_PROJECT,
+    MANAGED_COST_QUERY_SET_IDS.MONTHLY_SERVICE_ACCOUNT,
+    MANAGED_COST_QUERY_SET_IDS.MONTHLY_PRODUCT,
+];
+
+const defaultPeriod: Period = {
+    start: dayjs.utc().subtract(5, 'month').startOf('month').format(),
+    end: dayjs.utc().endOf('month').format(),
+};
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/cost-analysis/config.ts:21]: nitpick: there's `getInitialDates()` in cost-explorer/lib/helper.ts !

**Review by WANZARGEN — COMMENTED** [review]
Oh, the managed cost query set has been added. 
That makes sense to me! 
However, I've added a few comments regarding the LNB related aspects.

File: apps/web/src/services/cost-explorer/cost-analysis/config.ts (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/services/cost-explorer/cost-analysis/config.ts:21]: Thanks!

File: apps/web/src/services/cost-explorer/CostExplorerLNB.vue [hunk 2]
```diff
@@ -43,6 +43,44 @@ const state = reactive({
             },
         ], store.getters['user/pagePermissionList']),
     ]),
+    costAnalysisMenuSet: computed<LNBMenu[]>(() => [
+        { type: MENU_ITEM_TYPE.FAVORITE_ONLY },
+        {
+            type: MENU_ITEM_TYPE.TOP_TITLE,
+            label: 'Cost Analysis',
```
[the remark this answers is not in the document]
Comment (piggggggggy) [reply apps/web/src/services/cost-explorer/CostExplorerLNB.vue:50]: Oh.. I missed it, thanks!

File: apps/web/src/services/cost-explorer/store/cost-query-store.ts [hunk 3]
```diff
@@ -6,40 +6,41 @@ import { store } from '@/store';
...
-    costQueryList: CostQuerySetModel[];
+    costQuerySetList: CostQuerySetModel[];
     dataSourceList: string[];
-    selectedQueryId?: string;
+    selectedQuerySetId?: string;
     selectedDataSource?: string;
 }
 
-export const useCostAnalysisLNBStore = defineStore('cost-analysis-l-n-b', {
+export const useCostQueryStore = defineStore('cost-query', {
```
[the remark this answers is not in the document]
Comment (piggggggggy) [reply apps/web/src/services/cost-explorer/store/cost-query-store.ts:19]: OMG..

File: apps/web/src/services/cost-explorer/cost-analysis/config.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import { GRANULARITY, GROUP_BY } from '@/services/cost-explorer/lib/config';
+import type { CostQuerySetModel, Period } from '@/services/cost-explorer/type';
+
+export const MANAGED_COST_QUERY_SET_IDS = {
+    MONTHLY_PROJECT: 'Monthly cost by project',
+    MONTHLY_SERVICE_ACCOUNT: 'Monthly cost by service account',
+    MONTHLY_PRODUCT: 'Monthly cost by product',
+} as const;
+
+export const managedCostQuertSetIdList: string[] = [
```
Comment (sulmoJ) [inline apps/web/src/services/cost-explorer/cost-analysis/config.ts:12]: Typo !

**Review by WANZARGEN — APPROVED** [review]
LGTM!

**Pull request merged**